src/libs/API.js

```diff
@@ -179,13 +179,7 @@ Network.registerResponseHandler((queuedRequest, response) => {
         return;
     }
 
-    if (response.jsonCode === 405 || response.jsonCode === 404 || response.jsonCode === 402) {
-        // IOU Split & Request money transactions failed, due to invalid amount(405), unable to split(404), invalid phone number(402)
-        // It's a failure, so reject the queued request
-        queuedRequest.reject(response);
-        return;
-    }
```

Comment:
There may be consequences of removing this code for some API that are wrapped in a `catch()`, but it will be too much work to look at all API methods using a catch and find out whether it throws these code. I'm only going the fix the ones that are explicitly being handled.